transform/gc.go

```diff
@@ -186,11 +186,25 @@ func MakeGCStackSlots(mod llvm.Module) bool {
 				// be optimized if needed.
 			}
 
-			if ptr := stripPointerCasts(ptr); !ptr.IsAAllocaInst().IsNil() {
+			ptrStripped := stripPointerCasts(ptr)
+			if !ptrStripped.IsAAllocaInst().IsNil() {
 				// Allocas don't need to be tracked because they are allocated
 				// on the C stack which is scanned separately.
 				continue
 			}
+			if !ptrStripped.IsACallInst().IsNil() {
```

Comment:
Is this change still needed with https://github.com/tinygo-org/tinygo/pull/3395 ?

Reply:
No, I'll rebase it once #3395 is in.

Reply:
#3395 has been merged, so this can be rebased now.